emsdk_manifest.json

```diff
@@ -346,6 +346,54 @@
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
 
+  {
```

Comment:
Can we delete the corresponding part with node 16 above this line then?

Reply:
We normally leave the old stuff around so that folks can still install it if they want.